nkms/network/node.py

```diff
@@ -38,25 +38,36 @@ def find_ursula(self, id, offer=None):
 
     def reencrypt(self, work_order):
         ursula_rest_response = self.send_work_order_payload_to_ursula(work_order)
-        cfrags = RepeatingBytestringSplitter((CapsuleFrag, CFRAG_LENGTH))(ursula_rest_response.content)
+        cfrags = BytestringSplitter((CapsuleFrag, CFRAG_LENGTH)).repeat(ursula_rest_response.content)
```

Comment:
Do we have public-facing documentation on exactly what `.repeat()` does? Perhaps it should be in the BytestringSplitter repo?

Reply:
Yeah, true story.

Reply:
See https://github.com/nucypher/byteStringSplitter/issues/1